js-plot/vega-spec.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Do we still want log scale here? If user has a big initial guess, we will again have a problem with the plot.

Reply:
Could not get log scale to work together with layers and zoom/pan.

Initial guess is still hardcoded, so if the user changes it he/she should also change the start+stop .